lib/src/events.dart

```diff
@@ -13,8 +13,14 @@ class Emitter {
 
   Stream<Event> operator[] (type) => _evtController.stream.where((e) => e.type == type);
 
+  Stream<T> getEventStream<T extends Event>(String type) {
```

Comment:
Is there a reason this is being used instead of `operator[]`? Should `operator[]` be deprecated?

Reply:
as far as I could tell, there wasn't a way to make the `operator[]` approach strong-mode compliant because it returns different types of event streams based on the key. Adding a generic method was how I worked around that

Reply:
And yeah the `operator[]` should probably be deprecated... but this package is effectively deprecated so I didn't bother